package.json

```
@@ -215,5 +208,19 @@
   "eslintIgnore": [
     "packages/**/test",
     "packages/**/dist"
+  ],
+  "workspaces": [
+    "./packages/arcgis-rest-types",
+    "./packages/arcgis-rest-request",
+    "./packages/arcgis-rest-auth",
+    "./packages/arcgis-rest-demographics",
+    "./packages/arcgis-rest-feature-layer",
+    "./packages/arcgis-rest-geocoding",
+    "./packages/arcgis-rest-portal",
+    "./packages/arcgis-rest-request",
+    "./packages/arcgis-rest-routing",
+    "./packages/arcgis-rest-service-admin",
+    "./packages/arcgis-rest-fetch",
```

Can we use a wildcard here?
```suggestion
    "./packages/*",
```

---

@noahmulfinger you actually cannot because you need to compile things in dependency order. If you try to build `./packages/arcgis-rest-routing` before `./packages/arcgis-rest-request` you might get an error.

---

@patrickarlt Ah okay. Well in that case, line 220 can be removed because its a duplicated entry of `arcgis-rest-request`

---

If we switch to https://www.npmjs.com/package/ultra-runner (#741) we can use the `--build` option but OOTB `npm run build --ws` (run `build`) in all workspaces might fail by default.